docs/howto/serialize_poro.md

```diff
@@ -42,4 +42,32 @@ end
 
 The default serializer would be `MyModelSerializer`.
 
+*IMPORTANT*: There is a surprising behavior (bug) in the current implementation of ActiveModelSerializers::Model that
+prevents an accessor from modifying attributes on the instance.  The fix for this bug
+is a breaking change, so we have made an opt-in configuration.
```

Comment:
Do we agree that 1.0 can have as many breaking changes as required? I think this method is a nasty duct tape that we should remove.